events/admin.py

```diff
@@ -40,6 +41,8 @@ class EventAdmin(admin.ModelAdmin):
         'turnover',
         'absolute_price_change',
         'price_change',
+        'created_date',
+        'created_by',
```

Comment:
I've just added existed fields to visible list in Admin panel. I didn't create any new fields. Now user can add new event but can't modify it. Only staff can edit events (maybe even delete them).